tests/test_decoding.py

```diff
@@ -888,7 +888,7 @@ def test_decode_request_end_of_pdu_field(self) -> None:
         diag_layer._resolve_odxlinks(odxlinks)
         diag_layer._finalize_init(odxlinks)
 
-        coded_message = bytes([0x12, 0x34, 0x34])
+        coded_message = bytes([0x12, 0x34, 0x54])
```

Comment:
Why?

Reply:
I wanted the two items of the END-OF-PDU fields be different from each other. the second nibble of these bytes is captured by `struct_param_1` which is a MATCHING-REQUEST-PARAM and can not be changed for that reason...